src/assets/styles/overrides.scss

```diff
@@ -67,3 +67,16 @@
 .v-application--is-ltr .v-list-item__icon:first-child {
   margin-right: 1rem;
 }
+
+// Error Colours
```

Comment:
PS I don't think Storyboook (React?) is correctly importing our overrides stylesheet. I noticed one instance where our v-btn's styles were not being used.

(This is a future problem to fix.)